azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/header/InfoHeaderView.kt

```diff
@@ -64,6 +68,6 @@ internal class InfoHeaderView : ConstraintLayout {
     }
 
     private fun displayParticipantList() {
-        infoHeaderViewModel.displayParticipantList()
+        displayParticipantListCallback()
```

Comment:
Same thing here - this part of the code hasn't changed for a while, which means no new specs around 1displayParticipantsImageButton.setOnClickListener`, so can we remove `displayParticipantList()` and have the OnClickListener call `displayParticipantListCallback()` directly